### PR TITLE
Added Direct Link to Codespace

### LINKED ISSUE #43979: Add new codespace link to docs
Right now https://github.com/nextcloud/server/blob/master/.devcontainer/README.md just points to the docs of codespaces.

However, it would be nice to add the following link to directly spin up a new instance:
https://github.com/codespaces/new?hide_repo_select=true&ref=master&repo=60243197&skip_quickstart=true

Edit: also would be nice to note that it takes 5+ minutes to build

### PR DESCRIPTION
**Pull request by RajeevDash007**
* Resolves: #43979

## Summary
added the following link to directly spin up a new instance for better assistance to developer:
https://github.com/codespaces/new?hide_repo_select=true&ref=master&repo=60243197&skip_quickstart=true

### CODE REVIEW

**Comment by RajeevDash007** [comment]
@joshtrichards Please Review this PR

**Comment by RajeevDash007** [comment]
@RayBB please review this PR

**Comment by RajeevDash007** [comment]
@Antreesy please review this

**Comment by Antreesy** [comment]
I'm lacking knowledge for that. Please align with issue opener first

**Comment by RayBB** [comment]
The link you added is good. Adding a note that it takes 5+ minutes (maybe someone should time it)? Would also be valuable.

And you probably should remove those changes to that htaccess file.

File: .htaccess [hunk 1]
```diff
@@ -110,3 +110,7 @@
 
 AddDefaultCharset utf-8
 Options -Indexes
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
```
Comment (skjnldsv) [inline .htaccess:113]: Please drop this change

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: .htaccess (hunk 1 above)
Comment (RajeevDash007) [reply .htaccess]: Changes done

**Comment by RajeevDash007** [comment]
> The link you added is good. Adding a note that it takes 5+ minutes (maybe someone should time it)? Would also be valuable.
> 
> And you probably should remove those changes to that htaccess file.

Chages Done 👍

**Comment by skjnldsv** [comment]
> > The link you added is good. Adding a note that it takes 5+ minutes (maybe someone should time it)? Would also be valuable.
> > And you probably should remove those changes to that htaccess file.
> 
> Changes Done 👍

Please clean the commits history then :)
- Use [conventionalcommits](https://www.conventionalcommits.org/) message syntax
- Only have one commit that describe the change

**Comment by RajeevDash007** [comment]
Okay sure

**Pull request closed without merging**